src/wled/models.py

```diff
@@ -605,8 +605,12 @@ def from_dict(
         entries = [
             PlaylistEntry(
                 entry_id=entry_id,
-                duration=entries_durations[entry_id],
-                transition=entries_transitions[entry_id],
+                duration=entries_durations[entry_id]
+                if isinstance(entries_durations, list)
+                else entries_durations,
+                transition=entries_transitions[entry_id]
+                if isinstance(entries_transitions, list)
+                else entries_transitions,
```

Comment:
Please do not use inline ternaries that span multiple lines.

Reply:
Fixed. I also thought about adding default values if the value is empty. For duration it's easy - 100 (10 seconds), but for transition time it's a bit complicated since as I understand, it uses transition value from current state right before loading the playlist - https://github.com/Aircoookie/WLED/blob/main/wled00/playlist.cpp#L91.